sunscreen_compiler_common/src/context.rs

```diff
@@ -98,6 +101,20 @@ where
     }
 }
 
+impl<O> PartialEq for FrontendCompilation<O>
```

Comment:
This is really for testing, as graph isomorphism is generally an NP-complete problem. Add a comment as such.

petgraph uses the VF2 algortithm, which is minimally O(V^2) and O(V!*V) in the worst case (lol).